### PR TITLE
Made `IMPORT INTO` takes table offline more prominent

### LINKED ISSUE #11123: import: make it more prominent that `import into` takes the table offline
Michael Wang (mwang1026) commented:

https://www.cockroachlabs.com/docs/v21.1/import-into.html

We make note of this, but customers constantly run into this as a big surprise

We should also update \h help text in the SQL shell

### PR DESCRIPTION
**Pull request by kathancox**
Closes #11123 

Adjusted the positioning of the note letting users know that `IMPORT INTO` will take the table offline while the data is imported into the table. 

- Added this to the considerations section 20.2, 21.1, 21.2
- Added a note on this right above the examples
- Adjusted some of the surrounding text into body text rather note boxes to emphasize this notice relative to importance of the other information

### CODE REVIEW

**Comment by mwang1026** [comment]
cc @livlobo for review

**Review by livlobo — APPROVED** [review]
LGTM w.r.t. making it prominent.

Do we use the terminology The table is public again to mean that it is back online?

File: _includes/v21.1/import-into-offline.md
Comment (rmloveland) [inline _includes/v21.1/import-into-offline.md:1]: tbh I feel like this creeps closer to a warning to me than just "info".  meta: wish we had the middle ground "yellow caution note" for these situations

File: v21.1/import-into.md
Comment (rmloveland) [inline v21.1/import-into.md:13]: It's good that we're adding this note, but I wonder If 5 times on this page feels a bit on the nose stylistically? for me as a reader it's a bit much.  your call obv but my opinion is that if it's at the top under "Considerations" and at the bottom under "Known Limitations" we have definitely let the reader know.  if anything there is quite a bit of overlap between those two sections on this page and I'm wondering if we can consolidate.  At the end of the day it's more words to read that are saying the same things

also if it is technically a "known limitation" is there a GH issue we can link the user to so they know when/if this is lifted?  if not, and this is Just How It Works Now (tm) it's a shame we are deprecating `IMPORT TABLE` in favor of this which apparently does not have this limitation?  or am I wrong about that?  (Sorry to annoy but I'm reading this with my "skeptical user" hat on and feeling like I'm losing something)

File: v21.1/import-into.md
Comment (rmloveland) [inline v21.1/import-into.md:166]: from style POV this is probably the one I like the least tbh but if we do have to keep it is there a way to make every instance on this page use the include?  tons o repetition

**Review by rmloveland — APPROVED** [review]
LGTM but had comments - I would strongly prefer consolidation and fewer repetitions but your call obv.  Nothing blocking merge 

Separate issue and not something that can be addressed in this very topical PR but as a user I would like some clarity around how this interacts with the deprecation of `IMPORT TABLE` which, if it doesn't have these limitations (which it apparently does not according the `IMPORT` docs), makes me feel as a user like I'm losing something but not getting anything, and it isn't clear why - there is no explanation on the `IMPORT` page other than "this is happening".  If the response is "actually that's not accurate" or "please file a separate docs issue" that is fine, please let me know :-)

sorry, I am one of those annoying "why are you moving my cheese" users

**Pull request merged**
